src/pages/NewShoeForm.js

```diff
@@ -1,24 +1,64 @@
 import React, { useState } from 'react';
 
+// Airtable stuff
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
 function NewShoeForm() {
   const [location, setLocation] = useState('');
   const [bin, setBin] = useState('');
   const [part, setPart] = useState('');
+  const [partDescription, setPartDescription] = useState('');
   const [quantity, setQuantity] = useState('');
 
+  function clear() {
+    setLocation('');
+    setBin('');
+    setPart('');
+    setPartDescription('');
+    setQuantity('');
+  }
+
   const inputForm = (
     <form
       onSubmit={(evt) => {
         evt.preventDefault();
-        console.log(`Location: ${location}`);
-        console.log(`Bin Name: ${bin}`);
-        console.log(`Part Name: ${part}`);
-        console.log(`Quantity: ${quantity}`);
+        const quantityToInt = parseInt(quantity, 10);
+        if (Number.isNaN(quantityToInt)) { // invalid quantity input
```

Comment:
Would it be possible to have your error make a change on the front end (so instead of just logging, you write some kind of text on screen to indicate that you should be inputting a number). Ideally be specific, instead of just "Invalid input," say "Quantity needs to be a number" or something. In addition, you can enforce numeric input on the form: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number